app/controllers/photos_controller.rb

```diff
@@ -8,6 +8,7 @@ def find_album
 
   def find_photo
     @photo = Photo.find(params[:id])
+    @photo.view += 1;
```

Comment:
maybe the view should not be incremented each time the photo is found, but the only place I can think of the place is in show controller